.travis.yml

```diff
@@ -9,13 +9,26 @@ perl:
    - "5.18"
    - "5.16"
    - "5.14"
+   - "5.12"
+   - "5.10"
+   - "5.8"
 
 install:
+   - TEST_PERL="$PERLBREW_PERL"
+   - DZIL_PERL=`perl -e'eval"use 5.014";print $@?"5.14":$ENV{PERLBREW_PERL}'`
    - unset PERL_CPANM_OPT
+   - perlbrew use $DZIL_PERL
    - cpanm --mirror http://cpan.stratopan.com --quiet --notest Dist::Zilla
-   - dzil authordeps | cpanm --mirror http://cpan.stratopan.com --quiet --notest
-   - dzil listdeps   | cpanm --mirror http://cpan.stratopan.com --quiet --notest
-
-script:
+   - dzil authordeps |tee deps| cpanm --mirror http://cpan.stratopan.com --quiet --notest
+   - dzil listdeps   |tee -a deps| cpanm --mirror http://cpan.stratopan.com --quiet --notest
    - export HARNESS_OPTIONS='j:c'
-   - dzil smoke --release --author
+   - AUTHOR_TESTING=1
+   - RELEASE_TESTING=1
+   - AUTOMATED_TESTING=1
+   - dzil build --no-tgz
+   - perlbrew use $TEST_PERL
+   - grep -v Zilla deps|cpanm --mirror http://cpan.stratopan.com --quiet --notest
+   - cd Pinto*
+   - perl Build.PL
```

Comment:
Hi @jluis! I don't know the details about dzil breaking backward compatibility. But can you help me understand why you want to build this way instead of just using the `smoke` command? Thanks!